app.js

```diff
@@ -1,15 +1,15 @@
-var express = require('express')
-var app = express()
+var express = require('express');
```

Comment:
Unexpected var, use let or const instead. (no-var)
Expected empty line after require statement not followed by another require. (import/newline-after-import)
Unable to resolve path to module 'express'. (import/no-unresolved)